content/docs/quickstart.mdx

```diff
@@ -25,74 +25,63 @@ keywords:
 import ConfigDocker from '@site/content/examples/config/config.docker.yaml.md';
 import DockerCompose from '@site/content/examples/docker/basic.docker-compose.yml.md';
 
-In this quickstart guide, you'll run Pomerium Core with Docker containers.
+# Run Pomerium Core With Docker
 
-## Prerequisites
-
-- A configured [identity provider]
-- [Docker] and [Docker Compose]
+Run Pomerium Core with Docker containers in **under 5 minutes**.
 
-## Configure
+The Core quickstart uses Pomerium's **Hosted Authenticate Service**, but you can also configure a **self-hosted authenticate service** to integrate with Pomerium.
```

Comment:
add links

Reply:
Links aren't available in this PR. I just merged HA docs, so I'd have to merge into this one to link.